docs/api/01-overview.mdx

```diff
@@ -0,0 +1,88 @@
+---
+id: "overview"
+sidebar_position: 1
+title: "Overview"
+---
+
+import LinkPreview from "@site/src/components/LinkPreview";
+
+# The Sablier APIs
+
+Sablier relies on specific dependencies to source data or manage off-chain flows. All of these are either public or
+fully open-source, so feel free to roam around and suggest improvements or optimizations where needed.
+
+## Subgraphs and Indexers (GraphQL)
+
+As an alternative to reading data from the contracts or listening to onchain events, we use a set of subgraphs and
+indexers. These act as a middleware between the chain and our interfaces and allow for caching, formatting and querying
+data.
+
+### The Graph's Subgraphs
+
+[The Graph](https://thegraph.com/) has been the vendor of choice for the EVM space for the past few years. We've been
+integrating The Graph since 2019 with Sablier V1. For Sablier V2, we designed a special set of feature-oriented
+subgraphs used not only by us (in the official apps) but by 3rd party services (e.g. snapshot) and integrators alike.
+
+These subgraphs are hosted on The Graph Network, as well as on the The Graph Hosted Service. They can be used to query
+Sablier data from the official [endpoints](/api/subgraphs/endpoints). Every network has its own subgraph endpoints.
+
+Read more in the dedicated page about our `protocol` and `merkle` [subgraphs](/api/subgraphs/overview) or check out the
```

Comment:
```suggestion
Read more on the dedicated page about our `protocol` and `merkle` [subgraphs](/api/subgraphs/overview) or check out the
```

Everywhere it should be "Read more on the dedicated page" instead of "Read more in the dedicated page"